data_procesor/output_class.py

```diff
@@ -0,0 +1,56 @@
+import json
+
+from constans import emoji
+from utilities.log_class import LogClass
+
+
+class Output:
+    log: LogClass
+    flag: set()
+    duplicated_hour_table_jason = {"Time_table": {}}
+
+    def __init__(self, log):
+        self.log = log
+
+    def no_duplicates_final_output_to_list(self, time_table):
+        final_output_list = []
+        for hour in time_table:
+            reformed_hour_table = []
+            for minutes in hour[1]:
+                reformed_hour_table.append(f"{hour[0]}:{minutes}")
+            final_output_list.append(reformed_hour_table)
+        return final_output_list
+
+    def mixed_duplicates_final_output_to_json(self, time_table):
+        no_duplicates = []
+        for hour in time_table:
+            if hour[0] in self.flag:
+                self.duplicated_hour_table_jason["Time_table"][f"{hour[0]} hours"] = \
+                    {f"Bus - {index} at minute": minute for index, minute
+                     in enumerate(hour[1], start=1)}
+            else:
+                no_duplicates.append(hour)
+        if len(self.flag) < 3:
+            print(f"Without duplicates {emoji.TASK_SUCCSEEDED}:")
+            print(self.no_duplicates_final_output_to_list(no_duplicates))
```

Comment:
gali paaiskint sita if

Reply:
Na main mintis buvo pakeist printa is kazkiek duplicate kazkie ne i be visiskai nera bet dabar ziuriu kad nlb yra tolko. Deleted.

Reply:
*pakeistas